test/test_services_constructor.jl

```diff
@@ -96,6 +96,23 @@ end
     moi_tests(model, false, 1008, 0, 480, 216, 192, true)
 end
 
+@testset "Test Reserves from Thermal Standard UC with NonSpinningReserve" begin
```

Comment:
Please add a test in the `DecisionModel` test file that solves a problem with non-spinning reserves and also checks for the values.